### DISCUSSION TITLE
Persist after reboot?

### DISCUSSION THREAD

**Discussion by sblatnick** [tonistiigi/binfmt#87] [Q&A] [answered]
I'm adding multi-platform support to some images, where the docker buildx command was failing until I installed the emulators using:
```
docker run -it --rm --privileged tonistiigi/binfmt --install all
```
The problem I'm having is that after rebooting my system, I have to run this again before I can build again.

I am running Debian, so I could add a startup service in systemd, but that would re-install at each boot.  How can I get this to persist?

**Comment by tonistiigi** [comment, marked as answer]
https://github.com/tonistiigi/binfmt/issues/85

**Reply by sblatnick** [reply]
Looks like a startup service is the proposed solution.  Ok, thanks. It's pretty quick anyway.
